### PR TITLE
ref(core): Display used path when readFile and readFileSync errored

### PR DESCRIPTION
**Pull request by kamilogorek**
Fixes https://github.com/denoland/deno_std/issues/3166

### CODE REVIEW

File: runtime/ops/fs.rs [hunk 1]
```diff
@@ -2018,7 +2018,13 @@ fn op_readfile_sync(
...
     .borrow_mut::<PermissionsContainer>()
     .check_read(path, "Deno.readFileSync()")?;
-  Ok(std::fs::read(path)?.into())
+  let err_mapper = |err: Error| {
+    Error::new(
+      err.kind(),
+      format!("{}, readlink '{}'", err, path.display()),
+    )
+  };
+  Ok(std::fs::read(path).map_err(err_mapper)?.into())
```
Comment (dsherret) [inline runtime/ops/fs.rs:2027]: Maybe this should be done for op_readfile_text as well?

Comment (kamilogorek) [reply runtime/ops/fs.rs:2027]: Makes sense, done. Although I left the name as `readfile`, without `text` suffix, as its not important part of the IO.

Comment (aapoalas) [reply runtime/ops/fs.rs:2027]: There's an issue about FS errors in general that I opened at some point. stat functions, generic open, etc.

Should those be handled as well?

Comment (kamilogorek) [reply runtime/ops/fs.rs:2027]: @aapoalas from what I see `stat`, `open` and almost every other method is already covered. I might reactor this whole thing out though.

**Comment by aapoalas** [comment]
Closing this as the more complete followup was merged.

**Pull request closed without merging**